### ISSUE TITLE
New billing profile options

### ISSUE THREAD

**Issue by allinappliadmin** [Project60/org.project60.sepa#676]
It seems to not be possible to create a contribution for an organization. CiviSEPA calls the default individual profile with creates a new individual and also comes with unnecessary mandatory fields like 'country subdivision' or 'other first names'. (I'm not sure whether this is related to CiviCRM or the SEPA extension).

So this more or like a feature request to be able to use a custom profile with the desired fields and the ability to choose within individual / household or organization.

**Comment by bjendres** [comment]
You should be able to create a new SEPA mandate for any contact type in the backend. On my system that works, does it for you?

If you're talking about a payment processor, then you should raise the issue on the respective repository.

**Comment by allinappliadmin** [comment]
yes, I was talking about registering as a end user on the frontend. CiviSEPA is not controlling the profile as a payment processor? Which means that this is CiviCRM core issue, isn't it?

**Comment by bjendres** [comment]
Hm, I'm still not sure what you're referring to. Maybe a screenshot could help?

**Comment by allinappliadmin** [comment]
ok sorry. It's here: https://international-impact.com/fr/adhesion-particulier/
![image](https://github.com/Project60/org.project60.sepa/assets/62597730/f7b6bc99-4ca8-45ad-a277-620de572f9c7)

**Comment by allinappliadmin** [comment]
there's no way I can choose the profile that comes with the gateway and for example 'county' is mandatory but of no use in France. (just like Landkreis in Germany)

**Comment by bjendres** [comment]
Yes, this screenshot refers to a payment processor, probably this one: https://github.com/Project60/org.project60.sepapp

I'm really not using those the moment, but maybe if you post your problem there, somebody can help you. If not, maybe try https://civicrm.stackexchange.com

**Comment by allinappliadmin** [comment]
of course, sorry for mistaking, I'll post an issue there!
